packages/button/mwc-button-base.ts

```diff
@@ -108,6 +111,7 @@ export class ButtonBase extends LitElement {
       <button
           id="button"
           class="mdc-button ${classMap(this.getRenderClasses())}"
+          .value="${live(this.value) as unknown as string}"
```

Comment:
I don't think the `live` directive is doing much here, as the button value will not change on its own.

Let's remove it!

Reply:
sure, keeping `as unknown as string` to align with textfield/textarea. not sure if required here